Messenger/Base.lproj/MainMenu.xib

```diff
@@ -368,8 +368,8 @@
                                     <action selector="performMiniaturize:" target="-1" id="VwT-WD-YPe"/>
                                 </connections>
                             </menuItem>
-                            <menuItem title="Zoom" id="R4o-n2-Eq4">
-                                <modifierMask key="keyEquivalentModifierMask"/>
+                            <menuItem title="Zoom" keyEquivalent="z" id="R4o-n2-Eq4">
+                                <modifierMask key="keyEquivalentModifierMask" control="YES" command="YES"/>
```

Comment:
What's going on here? cmd-ctrl-z to make the window fullscreen?
(There are no standards apps with that binding AFAIK.)